<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>defect_classification</class>
 <widget class="QMainWindow" name="defect_classification">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>614</width>
    <height>592</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>缺陷分类</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QGraphicsView" name="graphicsView">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>40</y>
      <width>361</width>
      <height>301</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>40</y>
      <width>91</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>功能选择</string>
    </property>
   </widget>
   <widget class="QLabel" name="pre_classification">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>40</y>
      <width>121</width>
      <height>281</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">border: 1px solid black;</string>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="preclassify">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>150</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#C0C0C0;border-radius:5px;}QPushButton:hover{background:#FFD700;}</string>
    </property>
    <property name="text">
     <string>预分类</string>
    </property>
   </widget>
   <widget class="QPushButton" name="multi_task">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>180</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#C0C0C0;border-radius:5px;}QPushButton:hover{background:#FFD700;}</string>
    </property>
    <property name="text">
     <string>多任务分类</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>350</y>
      <width>81</width>
      <height>21</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Adobe 宋体 Std L</family>
      <pointsize>12</pointsize>
     </font>
    </property>
    <property name="text">
     <string>分类结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="saveResult">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>210</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#C0C0C0;border-radius:5px;}QPushButton:hover{background:#FFD700;}</string>
    </property>
    <property name="text">
     <string>保存结果</string>
    </property>
   </widget>
   <widget class="QPushButton" name="create_table">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>240</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#C0C0C0;border-radius:5px;}QPushButton:hover{background:#FFD700;}</string>
    </property>
    <property name="text">
     <string>生成报表</string>
    </property>
   </widget>
   <widget class="QPushButton" name="backMainWindow">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>80</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#008080;border-radius:5px;}QPushButton:hover{background:green;}</string>
    </property>
    <property name="text">
     <string>返回主页面</string>
    </property>
   </widget>
   <widget class="QPushButton" name="exitSystem">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>290</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#F7D674;border-radius:5px;}
QPushButton:hover{background:red;}</string>
    </property>
    <property name="text">
     <string>退出系统</string>
    </property>
   </widget>
   <widget class="QPushButton" name="selectImage">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>120</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#C0C0C0;border-radius:5px;}QPushButton:hover{background:#FFD700;}</string>
    </property>
    <property name="text">
     <string>选择图片</string>
    </property>
   </widget>
   <widget class="QPushButton" name="lastImg">
    <property name="geometry">
     <rect>
      <x>200</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#FFFACD;border-radius:5px;}QPushButton:hover{background:#D2691E;}</string>
    </property>
    <property name="text">
     <string>上一张</string>
    </property>
   </widget>
   <widget class="QPushButton" name="nextImg">
    <property name="geometry">
     <rect>
      <x>480</x>
      <y>350</y>
      <width>75</width>
      <height>23</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">QPushButton{background:#FFFACD;border-radius:5px;}QPushButton:hover{background:#D2691E;}</string>
    </property>
    <property name="text">
     <string>下一张</string>
    </property>
   </widget>
   <widget class="QTextEdit" name="resultShow">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>380</y>
      <width>511</width>
      <height>151</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">background-color:rgb(211,211,211);</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>614</width>
     <height>22</height>
    </rect>
   </property>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
 </widget>
 <resources/>
 <connections/>
</ui>
